src/main/configurations/Translate/Configuration_AndereZaak.xml

```diff
@@ -61,12 +61,35 @@
             <XmlIfPipe name="CheckForGetZgwAndereZaakResult"
                 xpathExpression="string-length(ZgwZaak) > 0"
                 >
-                <Forward name="then" path="CallPatchAndereZaak"/>
+                <Forward name="then" path="CheckVerwerkingssoortAttribute"/>
                 <Forward name="else" path="EXIT"/>
             </XmlIfPipe>
 
+            <XmlIfPipe name="CheckVerwerkingssoortAttribute"
+                getInputFromFixedValue="&lt;dummy/&gt;"
+                xpathExpression="$Verwerkingssoort = 'V'">
+                <Param name="Verwerkingssoort" sessionKey="Verwerkingssoort" defaultValue="''"/>
+                <Forward name="then" path="CallPatchAndereZaakDelete" />
+                <Forward name="else" path="CallPatchAndereZaak" />
+            </XmlIfPipe>
```

Comment:
Perhaps a XmlSwitchPipe would be a better fit here? Xpath to get verwerkingssoort and have forwards for "I", "T" and "V".
```suggestion
    <XmlSwitchPipe
        name="CheckVerwerkingssoortAttribute"
        getInputFromSessionKey="originalMessage"
        xpathExpression="*/@*:verwerkingssoort"
        notFoundForwardName="unimplemented"
        >
        <Forward name="I" path="CallIdentityAction" />
        <Forward name="V" path="CallDeleteAction" />
        <Forward name="T" path="CallAddAction" />
        <Forward name="unimplemented" path="UnrecognizedVerwerkingssoort_Error" />
    </XmlSwitchPipe>
```